### PR TITLE
Mes 3912 capture info on finalise outcome cat be

### PR DESCRIPTION
**Pull request by appleabs**
## Description
Added transmission component to finalise outcome page for cat-be and added tests for cat-b and cat-be for finalise outcome page.
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
image

### CODE REVIEW

File: src/pages/non-pass-finalisation/cat-be/__tests__/non-pass-finalisation.cat-be.page.spec.ts [hunk 1]
```diff
@@ -13,11 +13,22 @@ import { NonPassFinalisationViewDidEnter } from '../../non-pass-finalisation.act
...
+'../../../../components/test-finalisation/finalisation-header/finalisation-header';
+import { TransmissionComponent } from '../../../../components/test-finalisation/transmission/transmission';
+import { D255Yes, D255No, DebriefWitnessed, DebriefUnwitnessed } from
+'../../../../modules/tests/test-summary/test-summary.actions';
+import { CandidateChoseToProceedWithTestInWelsh, CandidateChoseToProceedWithTestInEnglish } from
+'../../../../modules/tests/communication-preferences/communication-preferences.actions';
+import { GearboxCategoryChanged } from '../../../../modules/tests/vehicle-details/vehicle-details.actions';
 
-describe('NonPassFinalisationCatBEPage', () => {
+fdescribe('NonPassFinalisationCatBEPage', () => {
```
Comment (ammarhaiderbjss) [inline src/pages/non-pass-finalisation/cat-be/__tests__/non-pass-finalisation.cat-be.page.spec.ts]: remove this focus?

File: src/pages/non-pass-finalisation/cat-be/non-pass-finalisation.cat-be.page.ts [hunk 2]
```diff
@@ -152,6 +160,24 @@ export class NonPassFinalisationCatBEPage extends BasePageComponent implements O
...
       ),
+      transmission$: currentTest$.pipe(
+        select(getVehicleDetails),
+        select(getGearboxCategory),
+      ),
+      transmissionAutomaticRadioChecked$: currentTest$.pipe(
+        select(getVehicleDetails),
+        map(isAutomatic),
+        tap((val) => {
+          if (val) this.form.controls['transmissionCtrl'].setValue('Automatic');
```
Comment (ammarhaiderbjss) [inline src/pages/non-pass-finalisation/cat-be/non-pass-finalisation.cat-be.page.ts:171]: Will this auto-populate these values on page load? I think we changed this recently so the user had to explicitly input a value..

Comment (appleabs) [reply src/pages/non-pass-finalisation/cat-be/non-pass-finalisation.cat-be.page.ts:171]: This is how it is currently done on pass finalisation i believe.

Comment (ammarhaiderbjss) [reply src/pages/non-pass-finalisation/cat-be/non-pass-finalisation.cat-be.page.ts:171]: Ah this is for a fail so it doesn't matter :)

Comment (appleabs) [reply src/pages/non-pass-finalisation/cat-be/non-pass-finalisation.cat-be.page.ts:171]: ok :)

**Review by rossfauldsbjss — APPROVED** [review]
looks good

**Review by ammarhaiderbjss — APPROVED** [review]

**Pull request merged**